relentless/optimize/__init__.py

```diff
@@ -10,7 +10,8 @@
 
 """
 from .method import (Optimizer,
+                     LineSearch,
                      SteepestDescent,
-                     LineSearch)
+                     FixedStepDescent)
 
-from .objective import (ObjectiveFunction)
+from .objective import (ObjectiveFunction)
```

Comment:
Minor formatting things: import the classes from `method` in alphabetical order, and restore the missing newline character at the end of this file (that's what the red circle here means).